pkg/config/config.go

```diff
@@ -16,6 +16,11 @@ const (
 	configFileName = "odo"
 )
 
+// ConfigurationInfo holds all odo specific configurations
+type ConfigurationInfo struct {
+	UpdateNotification bool `json:"updatenotification"`
```

Comment:
Make this pointer to `bool` to make it possible to detect whatever it was set or not.